Topics/User_Experience/User_Research.md

```diff
@@ -0,0 +1,102 @@
+
+Name: Emily
+
+Age: 20
+
+Occupation: UTSG student
+
+Goals: Find a quiet and productive study space that is close to campus
+
+Needs: Comfortable seating, wifi, power outlets, quiet environment
+
+Pain Points: Limited availability of study spaces, noisy environments, uncomfortable seating
+
+
+2. User Testing: User testing involves observing and gathering feedback from users as they interact with your product. This can be done through various methods, such as in-person interviews, remote testing, or A/B testing. The goal of user testing is to identify usability issues, pain points, and opportunities for improvement.
+
+Example: We have finished our prototype for study space finder application, we need to conduct a user testing then.
+
+Step 1: Identify the testing goals: Before conducting user testing, you need to identify the goals I want to achieve. For example, I might want to test the ease of use of the app, the effectiveness of the search function, and the relevance of the search results.
+
+Step 2: Recruit participants: Next, I need to recruit participants for the user testing. To do this, I can reach out to your target user base, such as college students or working professionals who are looking for study spaces. I could also offer incentives, such as a gift card or free coffee, to encourage participation.
+
+Step 3: Create a testing plan: Once I have recruited participants, create a plan for the user testing. This should include a list of tasks for participants to complete, such as searching for a study space near a specific location, and a set of questions to gather feedback on the app's usability and functionality.
+
+Step 4: Conduct the user testing: With the plan in place, conduct the user testing. This can be done in-person or remotely, depending on our needs and resources. As participants interact with the app, observe their behavior and ask them to provide feedback on their experience. Take notes on any usability issues or pain points that arise.
+
+Step 5: Analyze the results: After the user testing is complete, analyze the results to identify common themes and areas for improvement. Look for patterns in the feedback, such as difficulty finding study spaces or confusion with the search function.
+
+Step 6: Make improvements: Based on the results of the user testing, make improvements to the app. This might involve redesigning the search function, adding new features, or improving the user interface.
+
+Step 7: Repeat the process: Finally, repeat the user testing process to test the effectiveness of the improvements we have made. This will help us identify any additional issues and ensure that our app is meeting the needs of our users.
+
+3. Surveys and Questionnaires: Surveys and questionnaires are a way to gather quantitative data about user behavior and preferences. These can be useful for gathering feedback on specific features, understanding user needs, and measuring user satisfaction.
```

Comment:
I suggest putting surveys, questionnaires, interviews, and inquiries before user testing. You want to learn and specify the user's needs before the user testing. Changing a survey is much easier and cheaper than changing your app for user testing.